src/test/integration/tests/basic.ts

```diff
@@ -290,7 +292,7 @@ export const tests: {[name: string] : SSRTest} = {
     stableSelectors: ['div'],
   },
 
-  'array with templates': {
+  'NodePart accepts array with templates': {
```

Comment:
Is there NodePart accepts template? Looks like that's after this, why?

Reply:
These were existing tests. I can reorder them.

Reply:
Put arrays (& skipped repeats) last.